pkg/rkenodeconfigserver/nodeserver.go

```diff
@@ -111,6 +113,10 @@ func (n *RKENodeConfigServer) nonWorkerConfig(ctx context.Context, cluster *v3.C
 	if rkeConfig == nil {
 		rkeConfig = &v3.RancherKubernetesEngineConfig{}
 	}
+	prefix := cluster.Spec.RancherKubernetesEngineConfig.PrefixPath
```

Comment:
can we extract this block - lines 116-119 - to its own method? it gets repeated a couple of times